docs/managed-database-connection.md

```diff
@@ -0,0 +1,43 @@
+# Managed GCP PostgreSQL database connection
+
+## Prerequisites
+
+To set up the database, create the following:
+
+* Cloud SQL Postgres instance on Google Cloud Platform with public IP assigned
+* Key for GCP Service Account with at least one of the following roles ([click here for more information](https://cloud.google.com/sql/docs/postgres/connect-external-app#4_if_required_by_your_authentication_method_create_a_service_account)):
+  - Cloud SQL Client
+  - Cloud SQL Editor
+  - Cloud SQL Admin
+
+## Install the Compass with managed GCP PostgreSQL database
+
+To install the Compass with GCP managed Postgres database, set `database.useEmbedded` value to `false` inside `./chart/compass/values.yaml` file, and fill those values:
+
+| Parameter | Description | Values | Default |
+| --- | --- | --- | --- |
+| `global.database.managedGCP.serviceAccountKey` | Specifies base64 encoded key with Google credentials | base64 encoded string | "" |
+| `global.database.managedGCP.instanceConnectionName` | Specifies instance connection name to GCP PostgreSQL database | string | "" |
+| `global.database.managedGCP.dbUser` | Specifies database username | base64 encoded string | "" |
+| `global.database.managedGCP.dbPassword` | Specifies password for database user | base64 encoded string | "" |
+| `global.database.managedGCP.dbName` | Specifies database name | string | "" |
+| `global.database.managedGCP.host` | Specifies cloudsql-proxy host | string | "localhost" |
+| `global.database.managedGCP.hostPort` | Specifies cloudsql-proxy port | string | "5432" |
+| `global.database.managedGCP.sslMode` | Specifies SSL connection mode | string | "" |
+
+To connect to managed database, we use [cloudsql-proxy](https://cloud.google.com/sql/docs/postgres/sql-proxy) provided by Google, which consumes `serviceAccountKey` and `instanceConnectionName` values.
+
+Into `serviceAccountKey` value put the key for GCP Service Account mentioned in prerequisites.
```

Comment:
You should mention that in the table above.

Reply:
done